Tools/autotest/param_metadata/param_parse.py

```diff
@@ -307,6 +307,21 @@ def is_number(numberString):
         return False
 
 
+def clean_param(param):
+    if (hasattr(param, "Values")):
+        valueList = param.Values.split(",")
+        new_valueList = []
+        for i in valueList:
+            (start, sep, end) = i.partition(":")
+            if sep != ":":
+                raise ValueError("Expected a colon sep")
```

Comment:
```suggestion
                raise ValueError("Expected a colon separator, got %s" % i)
```

Lets at least use real word's here that people can look up rather then an abbreviation, and ideally if we print the content that failed it should be a bit easier to debug.

Reply:
Done.  I think this is actually a "can't happen", which is why I was a bit lazy.